samples/ingestion/ingestion-client/Setup/guide.md

```diff
@@ -64,9 +64,7 @@ In this mode audio files will be downloaded and streamed from the Azure Function
 
 1. Higher Azure Function costs [A single type Azure Functions will handle the process for each file and run at least for half the audio length]
 2. 2x processing of audio files [i.e. a 10-min file is transcribed in 5mins]
-3. There is no availability of the Diarization and Sentiment features in real time
-4. There is also no downstream SQL post processing option
-5. You will need to use the [Real Time ARM Template](ArmTemplateRealtime.json) from the repository for this operating mode.
+3. You will need to use the [Real Time ARM Template](ArmTemplateRealtime.json) from the repository for this operating mode.
```

Comment:
Why did you remove 3. and 4.? they are not supported rn.

another bullet point would be that we only support WAV format for realtme right now, whereas batch supports other formats as well

Reply:
i am writing only what is supported. I added the bullet on formats.